tutorials/spinnaker-binary-auth/index.md

```diff
@@ -0,0 +1,702 @@
+---
+title: CI/CD with Spinnaker and Binary Authorization
+description: Creating attestations by integrating Cloud Build, Spinnaker, Cloud KMS and Binary Authorization to securely deploy Kubernetes applications
+author: damadei
+tags: Kubernetes, Spinnaker, CI/CD, DevOps, Binary Authorization, Cloud KMS, Cloud Build
+---
+
+Daniel Amadei | Customer Engineer | Google
+
+# Introduction
+
+This solution exemplifies how to perform continuous integration and continuous delivery to a Kubernetes cluster using Cloud Build for building our images, Spinnaker for Continuous Deployment and Binary Authorization with keys hosted inside Cloud Key Management Service (Cloud KMS) for protecting and attesting the procedence of our images being deployed to Kubernetes. 
+
+We will start by setting up all our environment and then building the solution step by step. We hope you enjoy the journey.
```

Comment:
[A couple general thoughts:
1. First, this is my first review! So let's make sure everything is working with the review cycle, and then I'm happy to review further.

2. Generally speaking in this kind of writing, it's fine to drop the "we"/"we will" and just tell them what to do and using the second person singular "you" vs plural "we". Also, present tense is fine.  Putting both pieces of feedback together the above sentence becomes: "Start by setting up your environment."
]

Reply:
Great, will follow that!